package.json

```diff
@@ -16,7 +16,7 @@
     "install": "cd client && yarn install && cd ../executable && yarn install && cd ../server && yarn install",
     "watch": "cd client && yarn watch",
     "rebuild": "cd client && yarn rebuild-client",
-    "develop": "concurrently \"yarn component-dev\" \"yarn start\" ",
+    "develop": "concurrently \"yarn component-dev\" \"yarn start\" \"yarn storybook\"",
```

Comment:
this doesn't look quite right @lilitkarapetyan 

It looks like it will run `yarn start` and `yarn watch` in the `client` folder (from `start-client` and `component-dev` respectively)